styles.css

```
@@ -0,0 +1,260 @@
+body{
+
+    margin: 0px;
+    font-family: 'Roboto';
+   
+}
+
+.box{
+    width: 100%;
+    height: 64px;
+    background-color: #4E5BA6;
+    
+
+}
```

```suggestion
body{
    margin: 0px;
    font-family: 'Roboto';
}

.box{
    width: 100%;
    height: 64px;
    background-color: #4E5BA6;
}
```

의미를 갖지 않는 개행은 생략해주시는 게 더 가독성을 높이는 방법이예요!